CHANGELOG.md

```diff
@@ -5,12 +5,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.1.2] - 2023-02-16
+
+## Changed
+
+- `CLType::Any` is not supported anymore.
```

Comment:
Maybe we can explain the reason. "Any causes problems to parse events in a generic way because of the lack of a length indicator". Or similar...

Reply:
Done